executor/src/task.rs

```diff
@@ -121,6 +124,9 @@ pub async fn run_task(task: TaskCall, js: crate::JsCallback) -> Result<TaskRespo
                 }
                 RuntimeHostVm::StorageGet(req) => {
                     let key = HexString(req.key().as_ref().to_vec());
+
+					keys.push(key.clone());
```

Comment:
you don't need this. storage_diff has all the keys

Reply:
for getters storage_diff is always empty